apps/web-nextjs/src/components/ui/breadcrumb.tsx

```
@@ -0,0 +1,113 @@
+}
+
+function BreadcrumbPage({ className, ...props }: React.ComponentProps<"span">) {
+  return (
+    <span
+      data-slot='breadcrumb-page'
+      role='link'
+      aria-disabled='true'
+      aria-current='page'
+      className={cn("text-foreground font-normal", className)}
+      {...props}
+    />
```

Biome (2.1.2)

[error] The HTML element with the interactive role 'link' is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like 'link'.
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)